models/ArticleManager.php

```diff
@@ -0,0 +1,245 @@
+<?php
+
+require_once("Model.php");
+class ArticleManager extends Model
+{
+// FONCTIONS POUR LE FRONTEND
+    
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ["require_once" statement detected. File manipulations are discouraged. Statement is not a function, no parentheses are required.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)